client/state/notices/selectors.js

```diff
@@ -17,7 +12,7 @@ import 'calypso/state/notices/init';
  * @returns {Array}        Notice objects as array
  */
 export const getNotices = createSelector(
-	( state ) => values( state.notices.items ),
```

Comment:
How did you audit this one? Is `notices.items` always present?

Reply:
It's always an object, it defaults to an empty object at the reducer default state: https://github.com/Automattic/wp-calypso/blob/9e69a131fbed1a6ad4d017bfefcbdd2919541785/client/state/notices/reducer.js#L13